docs/sandbox-permissions.rst

```diff
@@ -1,40 +1,94 @@
-.. [#f2] Giving network access also grants access to all host services listening on abstract Unix sockets (due to how network namespaces work), and these have no permission checks. This unfortunately affects e.g. the X server and the session bus which listens to abstract sockets by default. A secure distribution should disable these and just use regular sockets.
+One of Flatpak's main goals is to increase the security of desktop systems by isolating applications from one another. This is achieved using sandboxing and means that, by default, Flatpak applications have extremely limited access to the host environment. This includes:
 
+- No access to any host files except the runtime, the app and ``~/.var/app/$APPID``. Only the last of these is writable.
+- No access to the network.
+- No access to any device nodes (apart from ``/dev/null``, etc).
+- No access to processes outside the sandbox.
+- Limited syscalls.  For instance, apps can't use nonstandard network socket types or ptrace other processes.
+- Limited access to the session D-Bus instance - an app can only own its own name on the bus.
+- No access to host services like X11, system D-Bus, or PulseAudio.
+
+Most applications will need access to some of these resources in order to be useful. This is primarily done during the finishing build stage, which can be configured through the ``finish-args`` section of the manifest file (see :doc:`manifests`).
+
+Portals
+-------
+
+Portals have already been mentioned in the :doc:`introduction`. They are a framework for providing access to resources outside of the sandbox, including:
+
+- Opening files with a native file chooser dialog
+- Opening URIs
+- Printing
+- Showing notifications
+- Taking screenshots
+- Inhibiting the user session from ending, suspending, idling or getting switched away
+- Getting network status information
+
+In many cases, portals use a system component to implicitly ask the user for permission before granting access to a particular resource. For example, in the case of opening a file, the user's selection of a file using the file chooser dialog is interpreted as implicitly granting the application access to whatever file is chosen.
+
+This approach enables applications to avoid having to configure blanket access to large amounts of data or services and gives users control over what their applications have access to.
+
+Interface toolkits like GTK and Qt implement transparent support for portals, meaning that applications don't need to any additional work to use them (it is worth checking which portals each toolkit supports). Applications that aren't using a toolkit with support for portals can refer to the `xdg-desktop-portal API documentation <https://flatpak.github.io/xdg-desktop-portal/portal-docs.html>`_ for information on how to use them.
```

Comment:
Oh you mention Qt/Gtk here, might be worth adding the version since only Gtk3 and Qt5 have it.

Reply:
Fixed.